Cargo.toml

```diff
@@ -182,6 +182,7 @@ opentelemetry-otlp = { version = "0.11.0" }
 proptest = "1.1.0"
 rustup-macros = { path = "rustup-macros" }
 tempfile = "3.5"
+termcolor.workspace = true
```

Comment:
this and the other change in this file are in the reverse spot from each other.

`workspace.dependencies` needs to specify a version
`dependencies` needs to specify that it uses the workspace version.

Reply:
Damn! 😢